src/test/java/org/example/custom/map_test.java

```diff
@@ -0,0 +1,91 @@
+package org.example.custom;
+
+
+import org.assertj.core.internal.MapsBaseTest;
+import org.assertj.core.util.VisibleForTesting;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.TreeMap;
+
+import org.assertj.core.internal.Maps;
+import org.apache.commons.collections4.map.CaseInsensitiveMap;
+import static org.assertj.core.util.Arrays.array;
+import static org.assertj.core.data.MapEntry.entry;
+
+public class map_test extends MapsBaseTest {
```

Comment:
use proper naming convention (look at the other test classes if you are not sure)

Reply:
and please follow the contributing guide https://github.com/assertj/assertj-core/blob/main/CONTRIBUTING.md thanks!